assets/js/components/legacy-setup/wizard-step-search-console-property.js

```diff
@@ -29,7 +29,7 @@ import { Component } from '@wordpress/element';
 /**
  * Internal dependencies
  */
-import SearchConsole from '../../components/setup/search-console';
+import SearchConsole from '../../components/legacy-setup/search-console';
```

Comment:
This could now import from `./search-console`. A similar simplification can be made to a few other imports that import from the same `legacy-setup` directory which the importing file is already in.

Reply:
👌